grunt/babel.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
     options: {
       sourceMap: false,
       presets: ['babel-preset-es2015', 'react'],
-      plugins: ['transform-class-properties'],
+      plugins: ['transform-class-properties', ['transform-runtime', {
+        polyfill: false,
+        regenerator: true,
```

Comment:
`regenerator` do we really need to enable this feature?

Reply:
Not needed, when I started working I was using async/await but then I remove it. I will remove the regenerator config and the dependencies installed.